packages/firebase/index.d.ts

```diff
@@ -7080,13 +7080,69 @@ declare namespace firebase.firestore {
    *
    *   <ul>
    *     <li><code>debug</code> for the most verbose logging level, primarily for
-   *     dubugging.</li>
+   *     debugging.</li>
    *     <li><code>error</code> to log errors only.</li>
    *     <li><code>silent</code> to turn off logging.</li>
    *   </ul>
    */
   export function setLogLevel(logLevel: LogLevel): void;
 
+  /**
+   * Converter used by `withConverter()` to transform user objects into
+   * Firestore data.
+   *
+   * Using the converter allows you to specify generic type arguments when
+   * storing and retrieving objects from Firestore.
+   *
+   * @example
+   *  ```typescript
+   * class Post {
+   *   constructor(readonly title: string, readonly author: string) {}
+   *     byline(): string {
```

Comment:
It is late in the day, but it took me way too long to figure that these are two methods. Can you add a new line and use a common term for the method, such as `toString()`?